src/data/tools.json

```
@@ -1366,6 +1366,19 @@
     "description": "The open-source Shopify alternative.",
     "logo": "medusa.png",
     "tag": [ "development platform", "tool", "API", "open source"]
+  },
+  {
+  "id": 151,
+   "title": "NVM - Node Version Manager",
+   "url": "https://github.com/nvm-sh/nvm",
+   "description": "Open Source version manager for Linux and Mac",
+    "tag": ["tool", "open-source", "package manager"]
```

The phrase "tool" is a bit too general. I would recommend the following:

```suggestion
    "tag": ["nodej", "node package manager", "package manager", "nvm"]
```